### DISCUSSION TITLE
Latest version of Taiko is not installing latest chromium

### DISCUSSION THREAD

**Discussion by Swarna-umusic** [getgauge/taiko#2282] [Development]
Latest Taiko version-1.2.7 is installing outdated chromium version mentioned below

"chromium_revision": "823078",
 "chromium_version": "88.0.4314.0" which is outdated

Whereas latest chromium version is   
 "chromium_revision": "915453",
  "chromium_version": "95.0.4623.0"

**Comment by gulshankamboj** [comment]
I am also facing issues in gmail login due to chromium version being 88.0.4314.0. I'm finding ways to use different browser or else update the chromium version. Will update here once done. 
Edit: Set up the path in default.properties as 
TAIKO_BROWSER_PATH = /Applications/Google Chrome.app/Contents/MacOS/Google Chrome (this path should point to the binary file) 
Similarly, you can install latest chromium using brew installer and point the path using above env variable.

**Comment by sriv** [comment]
same as #2441 

The later versions of Chromium have certain changes that affect the way taiko works. @zabil has been investigating this. Until we figure out a way to do this, we may not be able to let taiko use the latest version.

**Reply by gulshankamboj** [reply]
Thanks @sriv It would be of a great help. Meanwhile I'm trying to setup the browser path to chrome and Firefox nightly (https://docs.taiko.dev/experimental_features/#firefox-support) and somehow it is not working. Getting this error :  
Error Message: Error: Failed to launch browser! spawn /Applications/Firefox Nightly.app EACCES

Also tried separately installing chromium using brew installer and setting up that path using TAIKO_BROWSER_PATH and got the same error as above. Is it due to some packages are missing which are required?

**Reply by zabil** [reply]
> Error Message: Error: Failed to launch browser! spawn /Applications/Firefox Nightly.app EACCES

You need to pass the path to the exact firefox binary executable which is usually inside the .app folder.

**Reply by gulshankamboj** [reply]
Thanks @zabil this worked for me. Able to use the latest chromium and chrome as well. But the issue I'm facing is with Gmail login and I'm still facing this error. It would be a great help if you can suggest me a workaround? I know it's not specific to Taiko. 
<img width="1439" alt="Screenshot 2022-02-16 at 4 48 07 PM" src="https://user-images.githubusercontent.com/82322100/154254241-771b89d0-0ba2-45ff-9613-c0e394db066a.png">

**Reply by gulshankamboj** [reply]
@zabil Found a old discussion on the same issue https://spectrum.chat/taiko/general/cannot-login-google-account-in-chromium~9a866e76-2bd7-44d8-99b2-e162dd937f3c
